### PR TITLE
Qa/combine prerelease regression

### PR DESCRIPTION
**Pull request by SatrioAjieWijaya**
penambahan file untuk handle prerelease+regression dan regression yang di trigger secara manual

### CODE REVIEW

**Review by el7cosmos — CHANGES_REQUESTED** [review]
yang `manual-reg.yml` gabungin ke `regression.yml` aja, terus `prerelease-reg.yml` gabung ke `prerelease.yml`

File: workflow-templates/prerelease.yml [hunk 1]
```diff
@@ -2,16 +2,19 @@ name: "Post-Integration: Prerelease"
 
 on:
   pull_request:
-    branches: [ $default-branch ]
```
Comment (el7cosmos) [inline workflow-templates/prerelease.yml]: Keep this 2 lines

File: workflow-templates/regression.yml [hunk 2]
```diff
@@ -47,17 +60,23 @@ jobs:
 
       # Set report portal parameters environment variables
       # https://docs.github.com/en/actions/reference/workflow-commands-for-github-actions#setting-an-environment-variable
-      - name: Set Report Portal Parameters
-        run: |
-          NOW=$(date +%d%m%Y)
-          echo "rp.description='Regression ${NOW} - Github Action #${GITHUB_RUN_NUMBER}'" >> $GITHUB_ENV
-          echo "rp.attributes=release:${NOW}" >> $GITHUB_ENV
+      # - name: Set Report Portal Parameters
```
Comment (el7cosmos) [inline workflow-templates/regression.yml]: Remove this commented out lines

File: workflow-templates/regression.yml [hunk 3]
```diff
@@ -47,17 +60,23 @@ jobs:
...
-          rp.endpoint: ${{ secrets.RP_ENDPOINT }}
-          rp.api.key: ${{ secrets.RP_API_KEY }}
+        run: |
+          mvn -pl ${PROJECT} verify -q \
+          -Dwebdriver.base.url=${STG_ENV} \
+          -Drp.endpoint=${{ secrets.RP_ENDPOINT }} \
+          -Drp.api.key=${{ secrets.RP_API_KEY }} \
+          -Drp.description="Regression ${NOW} - Github Action #${GITHUB_RUN_NUMBER}" \
+          -Drp.attributes=release:${NOW}
+        # env:
```
Comment (el7cosmos) [inline workflow-templates/regression.yml]: Remove this commented out lines

File: workflow-templates/prerelease.yml [hunk 4]
```diff
@@ -2,16 +2,19 @@ name: "Post-Integration: Prerelease"
...
 
 jobs:
+
   prerelease:
     runs-on: ubuntu-20.04
     if: github.event.pull_request.merged == true
     outputs:
       key: ${{ steps.prerelease-tag.outputs.key }}
     steps:
+      - run: echo head_ref is ${{ github.head_ref }} | cut -d'/' -f 2
```
Comment (el7cosmos) [inline workflow-templates/prerelease.yml]: Remove this

File: workflow-templates/prerelease.yml [hunk 5]
```diff
@@ -28,4 +31,95 @@ jobs:
       name: staging:${{ needs.prerelease.outputs.key }}
       url: https://${{ needs.prerelease.outputs.key }}.example.com
     steps:
-      - run: echo 'Deployed'
```
Comment (el7cosmos) [inline workflow-templates/prerelease.yml:31]: Keep this for now, let infra hooks into this

**Review by el7cosmos — CHANGES_REQUESTED** [review]

**Review by el7cosmos — APPROVED** [review]

**Comment by el7cosmos** [comment]
LGTM
@billyjulius bisa bantu review?

**Review by billyjulius — APPROVED** [review]

**Pull request merged**